src/Messenger/IndexationRequestHandler.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace JoliCode\Elastically\Messenger;
+
+use Elastica\Document;
+use JoliCode\Elastically\Client;
+use Psr\Log\NullLogger;
+use Symfony\Component\Messenger\Exception\UnrecoverableMessageHandlingException;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+abstract class IndexationRequestHandler implements MessageHandlerInterface
+{
+    const OP_INDEX = 'index';
+    const OP_DELETE = 'delete';
+    const OP_UPDATE = 'update';
+    const OP_CREATE = 'create';
+
+    const OPS = [
+        self::OP_INDEX,
+        self::OP_DELETE,
+        self::OP_UPDATE,
+        self::OP_CREATE,
+    ];
+
+    private $client;
+
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+
+        // Disable the logs for memory concerns
+        $this->client->setLogger(new NullLogger());
```

Comment:
This is very opinated and in a consumer it will remove it forever.
Another move would be to wrap the content of `__invoke()` in `a try {} finally {}` to remove it only during the execution of this handler

Reply:
The logger consume *a lot* of memory, specially in a worker mode.

Another way could be to "clear" the logger at each `__invoke`... I don't understand what a try catch could solve?

Reply:
It's not a try catch, it's a try finally :) 
It's an easy way to do something, and undo it whatever happen in the try (IE: the finally is always executed)

Reply:
This will have to be handled in another PR, created https://github.com/jolicode/elastically/issues/17 to reference this.